src/FxCop/Desktop.Analyzers/Core/DesktopAnalyzersResources.resx

```diff
@@ -126,4 +126,46 @@
   <data name="DoNotCatchCorruptedStateExceptionsMessage" xml:space="preserve">
     <value>Modify {0} to catch and handle a more specific set of exception type(s) than {1} or rethrow the exception. </value>
   </data>
+  <data name="DoNotUseDES" xml:space="preserve">
+    <value>Do not use insecure cryptographic algorithm DES.</value>
+  </data>
```

Comment:
Since we have two rules now, I think the title and description should be changed to reflect the change, RE: weak vs. broken algorithms, as well.